etc/env/example.env

```diff
@@ -2,13 +2,20 @@
 # - - -
 
 # site options
-CANDIG_MODULES=weavescope portainer consul traefik logging monitoring minio mc drs-server htsget-server igv-js wes-server chord-metadata datasets cnv_service rnaget candig_server federation-service authorization jupyter
+CANDIG_MODULES=tyk keycloak vault weavescope portainer consul traefik logging monitoring minio mc drs-server htsget-server igv-js wes-server chord-metadata datasets cnv_service rnaget candig_server federation-service authorization
```

Comment:
like logging and monitoring, auth is comprised of a collection of services. As such, it might make it easier to keep them under one library/docker-compose stack, such as in lib/auth.

Reply:
good point, thank you